nixpkgs/github.json

```diff
@@ -2,5 +2,5 @@
   "owner": "obsidiansystems",
   "repo": "nixpkgs",
   "rev": "ebb270dfebbdda4800b5f5067041e40489d9e76e",
-  "sha256": "1dp5mhca547baxxjnragkda23l5gv7npk8sqm8h8a0xpal8jimz4"
+  "sha256": "13b5a0qpxsrszkw94srhvy8y2kl50wqs2i22qcdv4hb00c9lgkv4"
```

Comment:
It's a bit surprising that this approach would result in a different hash; is there a way we can keep the hashes interoperable?

Reply:
I was surprised as well; Going to try to figure out why this is in a bit. I was working on <a href="https://github.com/haskell-trasa/trasa"> trasa
</a> when I made this change and nix complained about needing this hash.

Reply:
One hunch is that this file is generated using nix-prefetch-git, which would make it incompatible with my branch's usage of fetchNixpkgs, which avoids this entirely.

Reply:
Given a Git revision hash `<rev>`, you get the new SHA256 by running:
```bash
$ nix-prefetch-url "https://github.com/NixOS/nixpkgs/archive/<rev>.tar.gz"
```
The SHA256 will be printed as the last line of stdout.